tools/crm_resource.c

```diff
@@ -901,12 +967,10 @@ list_providers(const char *command, const char *spec, crm_exit_t *exit_code)
     if (pcmk__str_any_of(command, "--list-ocf-providers",
                         "--list-ocf-alternatives", NULL)) {
         rc = lrmd_conn->cmds->list_ocf_providers(lrmd_conn, optarg, &list);
-        rc = pcmk_legacy2rc(rc);
         text = "OCF providers";
 
     } else if (safe_str_eq("--list-standards", command)) {
         rc = lrmd_conn->cmds->list_standards(lrmd_conn, &list);
-        rc = pcmk_legacy2rc(rc);
```

Comment:
Looks like maybe you figured out the rc was different? But something still seems off, besides being in the wrong commit

Reply:
This should be taken care of in the next push.  It shouldn't have been here in the first place.